contracts/RelayHub.sol

```diff
@@ -238,23 +239,27 @@ contract RelayHub is IRelayHub {
 
     function recipientCallsAtomic(address from, address to, address relayAddr, bytes calldata encodedFunction, uint transactionFee, uint gasLimit, uint initialGas) external returns (bool) {
         // This function can only be called by RelayHub.
-        // In order to Revert the client's relayedCall if postRelayedCall reverts, we wrap them in one function.
+        // In order to Revert the client's relayedCall if preRelayedCall/postRelayedCall reverts, we wrap them in one function.
```

Comment:
This comment fix it not needed. If preRelayedCall reverts, the relayed call wasn't
Called yet, so there's nothing more to revert

Reply:
Yes, I meant not to run the relayed call in the first place